core/src/test/scala/org/apache/spark/SparkConfSuite.scala

```diff
@@ -354,6 +354,20 @@ class SparkConfSuite extends SparkFunSuite with LocalSparkContext with ResetSyst
     }
   }
 
+  test("SPARK-26998: SSL configuration not needed on executors") {
+    val conf = new SparkConf(false)
+    conf.validateSettings()
```

Comment:
This line can be removed as `validateSettings()` is enough to be called once in line 365.

Reply:
Ping. (In fact do you need to call this method at all in this test.)

Reply:
Yep, I'm dealing with multi-cluster Kafka delegation token. Thanks for pinging me, forgot this but now fixed.